testsuite/features/secondary/min_ssh_tunnel.feature

```diff
@@ -44,7 +44,6 @@ Feature: Register a Salt system to be managed via SSH tunnel
     And I click on "Install Selected Packages"
     And I click on "Confirm"
     Then I should see a "1 package install has been scheduled for" text
-    When I force picking pending events on "ssh_minion" if necessary
```

Comment:
Are you sure about removing this? Same with the SSH minion below.

Reply:
Yes, rhn_check does not exist on minions, it's trad only

Reply:
Yup